src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { Todo } from '../../types/Todo';
+import { FilterStatus } from '../../types/FilterStatus';
+import cn from 'classnames';
+
+interface Props {
```

Comment:
It's better to name this type something like `FooterProps`, so that the name describes this type the most precisely.